README.md

```diff
@@ -58,6 +58,7 @@ Below configurations can be put together with spark-defaults.conf or passed in t
 | spark.tispark.meta.reload_period_in_sec |  60 | Metastore reload period in seconds |
 | spark.tispark.plan.allow_agg_pushdown |  true | If allow aggregation pushdown (in case of busy TiKV nodes) |
 | spark.tispark.plan.allow_index_double_read |  false | If allow index double read (which might cause heavy pressure on TiKV) |
+| spark.tispark.plan.use_index_scan_first |  false | Use index prior to table scan (takes effect only when `allow_index_double_read` is set to true) |
```

Comment:
Actually what we want is finer grain control over which index to choose. We already has an option to enable/disable index. It will be redundant to have index_first option.